### ISSUE TITLE
CNV Caller: PennCNV

### ISSUE THREAD

**Issue by ngiangre** [NCBI-Hackathons/Global_Screening_Arrays#31]
We already know about this, but I'm adding it for completeness with the reference

http://genome.cshlp.org/content/17/11/1665.long

**Comment by ekarlins** [comment]
Good idea!  If we run into issues or limitations with PennCNV we can add it hear.
